File: yeongseong/3week/369게임.js
function solution(order) {
    var answer = 0;
    // 입력된 숫자를 문자열로 변환하고 각 자릿수를 순회하면서 검사합니다.
    for (let number of String(order)){
        // 3, 6, 9가 나올 때마다 answer 값을 1씩 증가시킵니다.
        if (number === '3'|| number==='6' || number==='9') answer++
    }
    // 3, 6, 9의 총 횟수를 반환
    return answer;
}

File: gang/week3/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String str = Integer.toString(order);   // int형 변수 order를 String형으로 변환
        char[] result = str.toCharArray();      // String 문자열을 문자 배열로 변환

        for (int i = 0; i < result.length; i++) {   // 배열 크기만큼 반복문 실행
            char num = result[i];  // 배열의 각 값을 char형 변수 num에 넣어서
            if (num == '3' || num == '6' || num == '9') {   // num의 값이 3,6,9인지 확인
                answer++;
            }
        }

        return answer;
    }
}